tools/diag/publish_index.py

```
@@ -2060,7 +2060,7 @@ def _as_text_preview(path: Path, max_bytes: int = MIRROR_TEXT_LIMIT) -> str:
         ]
         for info in infos[:200]:
             lines.append(
-                "- {0} (compressed {1:,} bytes → {2:,} bytes)".format(
+                "- {0} (compressed {1:,} bytes -> {2:,} bytes)".format(
                     info.filename, info.compress_size, info.file_size
                 )
```

Ruff (0.15.21)

[warning] Use implicit references for positional format fields

Remove explicit positional indices

(UP030)